webapp/src/js/services/enketo.js

```diff
@@ -65,15 +65,16 @@ angular.module('inboxServices').service('Enketo',
 
     const replaceJavarosaMediaWithLoaders = function(formDoc, formHtml) {
       formHtml.find('[data-media-src]').each(function() {
-        $(this)
+        const $img = $(this);
+        const lang = $img.attr('lang');
+        const active = $img.is('.active') ? 'active' : '';
+        $img
           .css('visibility', 'hidden')
-          .wrap('<div class="loader">');
+          .wrap(() => '<div class="loader ' + active + '" lang="' + lang + '"></div>');
```

Comment:
By copying the active class and lang the spinner is hidden for unselected languages.